core/flyout_base.js

```diff
@@ -380,6 +392,9 @@ Blockly.Flyout.prototype.setVisible = function(visible) {
 
   this.isVisible_ = visible;
   if (visibilityChanged) {
+    if (!this.autoClose) {
+      this.workspace_.recordDragTargets();
```

Comment:
Add a comment about why you had to add this.

Reply:
Do you have a suggestion for a useful comment?
I've been in this code for so long that the only thing I can think of is: "Non auto-close flyouts need to have their dragTarget recomputed" which is just describing the code exactly and doesn't add anything.

Reply:
added a comment